sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/cryptography/CryptographyAsyncClient.java

```diff
@@ -79,6 +79,9 @@
         if (!jsonWebKey.isValid()) {
             throw new IllegalArgumentException("Json Web Key is not valid");
         }
+        if (jsonWebKey.getId() == null) {
+            throw new IllegalArgumentException("Json Web Key's id property is not configured");
```

Comment:
Adding the check here, means LocalCryptographyClient will be required to provide an id too.
This seems like a requirement only for LocalKeyEncryptionKeyClient / LocalKeyEncryptionKeyAsyncClient